### PR TITLE
feat(object): added Partial alias for OptionalPart

### PR DESCRIPTION
**Pull request by mesqueeb**
## 🎁 Pull Request

* [x] Used a clear / meaningful title for this pull request
* [ ] Tested the changes in your own code (on your projects)
* [x] Added / Edited tests to reflect changes (`tst` folder)
* [x] Have read the **Contributing** part of the **Readme**
* [x] Passed `npm test`

#### What changes have you made?
* added Partial type

#### What tests have you updated?
* added Partial test in `tst/object`

#### Is there any breaking changes?

* [ ] Yes, I changed the public API & documented it
* [ ] Yes, I changed existing tests
* [ ] No,  I added to the public API & documented it
* [x] No,  I added to the existing tests
* [ ] I don't know

### CODE REVIEW

File: src/Object/Optional.ts [hunk 1]
```diff
@@ -31,7 +32,7 @@ type OptionalPart<O extends object, depth extends Depth> = {
 Make some fields of **`O`** optional (deeply or not)
 @param O to make optional
 @param K (?=`Key`) to choose fields
-@param depth (?=`'default'`) to do it deeply
```
Comment (mesqueeb) [inline src/Object/Optional.ts:34]: @pirix-gh I believe this was a typo?

File: src/Object/Partial.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+/**
+Partial is an alias for Optional with a simpler syntax. Just like the native `Partial` it makes a type's fields optional, but accepts one extra param to choose between 'flat' or 'deep'.
+@param O the type to make its fields optional
+@param depth (?=`'flat'`) to do it deep
+@returns [[Object]]
+@example Partial<MyType>
+@example Partial<MyType, 'deep'>
+ */
+export type Partial<O extends object, depth extends Depth = 'flat'> = OptionalPart<O, depth>
```
Comment (mesqueeb) [inline src/Object/Partial.ts:12]: Added in new file as per your request!

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [6.10.0](https://github.com/pirix-gh/ts-toolbelt/compare/v6.9.0...v6.10.0) (2020-07-06)
```
Comment (mesqueeb) [inline CHANGELOG.md:5]: @pirix-gh this was auto generated, I didn't touch anything!

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-toolbelt",
-  "version": "6.9.9",
+  "version": "6.10.0",
```
Comment (mesqueeb) [inline package.json:3]: @pirix-gh this was auto incremented, I didn't touch anything!

File: tst/Object.ts [hunk 5]
```diff
@@ -807,7 +807,7 @@ checks([
 ])
 
 // ---------------------------------------------------------------------------------------
-// OPTIONAL
+// OPTIONAL & PARTIAL
```
Comment (pirix-gh) [inline tst/Object.ts:810]: You must create a separate bundle of tests for `Partial`, does not need to be crazy, but must at least reflect `deep` and `flat` usages.

File: src/Object/Partial.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import {OptionalPart} from './Optional'
+import {Depth} from './_Internal'
+
+/**
+Make a type's fields optional. Choose to do it deeply or not.
+@param O the type to make its fields optional
+@param depth (?=`'flat'`) to do it deeply
+@returns [[Object]]
+@example Partial<MyType>
```
Comment (pirix-gh) [inline src/Object/Partial.ts:9]: Examples should follow this form:

```
@example
.```ts
import {...} from 'ts-toolbelt'

// this must be a self contained example
// that can we either copy pasted or
// that will be runnable on the playground
// (for when the new doc website arrives)
.```
```

Comment (pirix-gh) [reply src/Object/Partial.ts:9]: Don't leave the dot in front of .```

**Comment by mesqueeb** [comment]
Cool! I'll do it tomorrow :)

**Comment by pirix-gh** [comment]
@mesqueeb ?

**Comment by mesqueeb** [comment]
@pirix-gh sorry i suddenly got busy with some deadlines at work >< 
I’ll hope to make some time this weekend!!

**Comment by mesqueeb** [comment]
@pirix-gh I have added 2 extra commit implementing your requests.
Is there anything else I need to do ?

**Comment by pirix-gh** [comment]
That's it, thank you :tada:. I would be more than happy if you could help with the docs, even just a list of things that are unclear (no explanations needed), then I can work through it and ask you what was unclear if needed.

**Pull request merged**